tests/ci/cdk/app.py

```diff
@@ -50,8 +50,8 @@
 AwsLcGitHubAnalyticsStack(app, "aws-lc-ci-analytics", analytics_build_spec_file, env=env)
 # bm_framework_build_spec_file = "cdk/codebuild/bm_framework_omnibus.yaml"
 # BmFrameworkStack(app, "aws-lc-ci-bm-framework", bm_framework_build_spec_file, env=env)
-mac_arm_build_spec_file = "cdk/codebuild/github_ci_macos_m1_omnibus.yaml"
-AwsLcMacArmCIStack(app, "aws-lc-ci-macos-arm", mac_arm_build_spec_file, env=env)
+ec2_test_framework_build_spec_file = "cdk/codebuild/ec2_test_framework_omnibus.yaml"
```

Comment:
I can't remember, but I think we will need to remember to manually terminate this stack. It looks like the resources created in it are all unique I believe? Don't believe CDK will handle this for us in this instance.